vm-setup/roles/v1aX_integration_test/tasks/move.yml

```diff
@@ -0,0 +1,100 @@
+---
+  - name: Label BMO CRDs.
+    shell: 'kubectl label --overwrite crds baremetalhosts.metal3.io {{ item }} --overwrite '
+    with_items:
+       - clusterctl.cluster.x-k8s.io=""
+       - cluster.x-k8s.io/provider="metal3"
+    
+  - name: Obtain target cluster kubeconfig
+    shell: "kubectl get secrets {{ CLUSTER_NAME }}-kubeconfig -n {{ NAMESPACE }} -o json | jq -r '.data.value'| base64 -d > /tmp/target.yaml"
+    register: target_kubeconfig
+
+  - name: Pivoting test
+    delegate_to: "{{ CLUSTER_APIENDPOINT_IP }}"
+    vars:
+      ansible_user: "{{ IMAGE_USERNAME }}"
+      ansible_ssh_private_key_file: "{{ SSH_PRIVATE_KEY }}"
+      ansible_python_interpreter: /usr/bin/python3
+    become: yes
+    become_user: "{{ IMAGE_USERNAME }}"
+    block:
+      - name: Create namespace
+        k8s:
+          name: "{{ NAMESPACE }}"
+          kind: Namespace
+          state: present
+          
+      - name: Clone Baremetal Operator
+        git:
+          repo: 'https://github.com/metal3-io/baremetal-operator.git'
+          dest: "/home/{{ IMAGE_USERNAME }}/baremetal-operator"
+            
+      - name: Download kustomize
+        get_url:
+          url: https://raw.githubusercontent.com/kubernetes-sigs/kustomize/master/hack/install_kustomize.sh
+          dest: "/home/{{ IMAGE_USERNAME }}/baremetal-operator/"
+          mode: '655'
+        register: kustomize_manifest
+
+      # Install Kustomize
+      - name: Install kustomize
+        shell: "bash /home/{{ IMAGE_USERNAME }}/baremetal-operator/install_kustomize.sh"
+
+      # Replacing the name of the bridge
+      - name: Replace the Provisioning Interface in ironic_ci.env
```

Comment:
I tested with `lineinfile` and it worked pretty well. Maybe you better use the same here instead of `replace`?

```
  - name: something
    lineinfile:
      path: /path_to_file/ironic_ci.env
      regexp: '^PROVISIONING'
      line: "PROVISIONING_INTERFACE={{ IRONIC_ENDPOINT_BRIDGE }}"
```